deployment/helm/values.yaml

```diff
@@ -0,0 +1,67 @@
+
+replicaCount: 1
+
+image:
+  pullPolicy: IfNotPresent
+  httpGateway:
+    repository: vesoft/nebula-http-gateway
+    tag: v2
+  nebulaImporter:
+    repository: vesoft/nebula-importer
+    tag: v2
+  nebulaStudio:
+    repository: vesoft/nebula-graph-studio
+    tag: v3
+  nginx:
+    repository: nginx
+    tag: alpine
+
+imagePullSecrets: []
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 7001
+
+resources:
+  httpGateway:
+    limits:
+      cpu: 2
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 256Mi
```

Comment:
We usually recommend not to specify default resources and to leave this as a conscious
choice for the user. If set here, we'd better specify a lower value.

Reply:
For these resource limits/requests, I prefer to leave them empty to let the user define them.